ZenPacks/zenoss/OpenStackInfrastructure/api.py

```diff
@@ -47,7 +47,8 @@ def _runcommand(cmd):
             message = json.loads(stdout)['error']
         except Exception:
             message = stderr
-
+            
+        cmd[3] = '--api_key=***'
```

Comment:
This seems like it could cause confusing problems later if we use _runcommand for something else.  Can you make it loop over and check for the command args starting with --api_key=, at least?  (Ideally we'd mask the specific password, wherever it occurs, but that's more of a pain because that info is not currently passed to _runcommand.. I understand..